x/evidence/types/msgs_test.go

```diff
@@ -50,8 +51,8 @@ func TestMsgSubmitEvidence(t *testing.T) {
 	}
 
 	for i, tc := range testCases {
-		require.Equal(t, tc.msg.Route(), types.RouterKey, "unexpected result for tc #%d", i)
-		require.Equal(t, tc.msg.Type(), types.TypeMsgSubmitEvidence, "unexpected result for tc #%d", i)
+		require.Equal(t, tc.msg.(legacytx.LegacyMsg).Route(), types.RouterKey, "unexpected result for tc #%d", i)
+		require.Equal(t, tc.msg.(legacytx.LegacyMsg).Type(), types.TypeMsgSubmitEvidence, "unexpected result for tc #%d", i)
```

Comment:
why this is `LegacyMsg`?

Reply:
We moved `sdk.Msg` to `legacytx.LegacyMsg`. So in the SDK all msgs are still`legacytx.LegacyMsg`.